### ISSUE TITLE
Animated SVGs

### ISSUE THREAD

**Issue by cohen72** [SVGKit/SVGKit#90]
Is there a way to download an animated svg and play it in a UIView?

For example, take the following svg:

http://www.hypnobeast.com/spirals/Hypno_Beast_Spiral_1000x1000.svg

Any words of wisdom? 

Thanks!

**Comment by adamgit** [comment]
Right now - no way. No-one has even looked at implementing the "animation" part of the SVG spec.

If you put the time into reading the spec, and working out what needs to be done, I think it would be quite easy, and I could help you to decide how/where to change the main code. I get the impression you only have to change a handful of low-level methods and variables, and it all comes together like magic.

...but it seems almost no-one is using / needs animations yet (other platforms have poor support for them, and the main editors have poor support for them - e.g. Inkscape sucks at animating things :( ).

Personally: I have no use for it, all my projects have complex animations that interact with mainstream Apple classes, and so I couldn't use the SVG animation system anyway :(.

**Comment by adamgit** [comment]
PS: That is indeed a terrifying hypnobeast.

(I had a brief look at animation support recently - I think it would be easy to add. But as noted: I have no use for it personally, so you'd need to add it yourself)

**Comment by kenmadsen** [comment]
Has there been made any plans in regards to this issue?
I'm in a situation right now in which I could really need animation, however I'm currently not at a stage in life in which I can start an implementation my self. I could however compensate someone willing to implement this.

For SVG animation I'm currently using Flash and exporting to SVG using Thomas Byrne's SVG extension.

**Comment by adamgit** [comment]
I can advise, but no time to code right now. Perhaps put out a call (if you want to tweet something, @t_machine_org and I'll retweet it)

**Comment by anton-matosov** [comment]
Current demo app has some animated SVGs (e.g. Monkey from local/fast bucket). So I assume animated SVGs are now fully supported and this ticket can be closed.

**Comment by adamgit** [comment]
No, the app uses Apple's Quartz / CoreAnimation - there is zero support for 
SVG animation.
